templates/taxi/index.html

```diff
@@ -10,4 +10,6 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+
+  <p>You have visited this page {{ num_visits }}</p>
```

Comment:
"You have visited this page ... **times**."

Reply:
oops, i forget to change this.....